README.md

```diff
@@ -6,12 +6,19 @@ Support for running [ElasitcMQ](https://github.com/adamw/elasticmq) in tests.
 
 Installation
 ------------
+
 Add the following to your `project/plugins.sbt` file:
 ```
-addSbtPlugin("com.localytics" % "sbt-sqs" % "0.4.2")
+addSbtPlugin("com.localytics" % "sbt-sqs" % "0.5.0")
```

Comment:
Maybe we should bump to `1.0.0` since this may be a breaking change for some folks?